### ISSUE TITLE
Ability to have month/day grid like in react-yearly-calendar

### ISSUE THREAD

**Issue by alexanderwhatley** [fullcalendar/fullcalendar#5119]
There is a library called react-yearly-calendar (https://github.com/BelkaLab/react-yearly-calendar) which shows the days of the yearly in a nice grid format, where the months are the rows and the days are the columns. However, this library has very little functionality and is thus not suited for the purposes of my company. Is there an example of the fullcalendar, using the premium vertical resource view, that is able to do something like this?

**Comment by acerix** [comment]
There is no built-in view like that, but I see how it could be useful as a new view if you mean something like this example:

https://raw.githubusercontent.com/BelkaLab/react-yearly-calendar/master/examples/screen.png

**Comment by alexanderwhatley** [comment]
Yeah, I think this view would be very useful. It's easier to events vertically this way, compared to the currently proposed yearly views, which seem like they center mainly around a grid of smaller calendars.

**Comment by sanjayshingadia** [comment]
Do we have a date when this will be released?

**Comment by TommySharpNZ** [comment]
This would be a great addition for planning leave and events.
